torchci/lib/bot/drciBot.ts

```diff
@@ -35,41 +35,45 @@ export default function drciBot(app: Probot): void {
     const repo = context.payload.repository.name;
     const owner = context.payload.repository.owner.login;
     
-    console.log(pr_owner);
-    if (pr_owner in possibleUsers) {
-      const existingValidationCommentData = await getDrciComment(
-        context,
-        prNum,
+    context.log(pr_owner);
+
+    if (possibleUsers.indexOf(pr_owner) == -1) {
+      context.log("did not make a comment")
+      return;
+    }
+
+    const existingValidationCommentData = await getDrciComment(
+      context,
+      prNum,
+      owner,
+      repo
+    );
+    const existingValidationCommentID = existingValidationCommentData[0];
```

Comment:
usually, it's better to return an object instead of an array of values. That way you can do something like
`const {existingValidationCommentID, existingValidationComment} = existingValidationCommentData;`

Reply:
also its just a little easier to understand

Reply:
just fixed this!